flex-linux-setup/flex_linux_setup/flex_setup.py

```
@@ -309,26 +309,27 @@ def set_app_versions_from_arguments(brgsp):
     """
     version = app_versions['NODE_MODULES_BRANCH'].replace('/', '-')
 
-    if version.startswith('v') or version == 'nightly':
+    if version.startswith('v'):
         release_tag = version
     else:
-        release_tag = 'admin-ui-' + version
+        release_tag = 'nightly'
+        version = 'nightly'
 
     url = '{0}/{1}/admin-ui-{2}-built.tar.gz'.format(admin_ui_releases_base_url, release_tag, version)
```

Ruff (0.16.0)

[warning] Use implicit references for positional format fields

Remove explicit positional indices

(UP030)